dotcom-rendering/src/components/DiscussionApps.importable.tsx

```diff
@@ -31,10 +77,8 @@ const onRecommend = async (commentId: string): Promise<boolean> => {
 		);
 };
 
-const addUsername = async (): Promise<Result<string, true>> => {
-	console.log('addUsername');
-	return { kind: 'error', error: 'ApiError' };
-};
+const addUsername = async (): Promise<Result<string, true>> =>
+	error('Unimplemented');
```

Comment:
Maybe in a follow up we want to distinguish between an `AppsReader` and `WebReader`, the former never having a `addUsername` method…